test/specs/spaniel-observer.spec.js

```diff
@@ -203,4 +204,32 @@ describe('SpanielObserver', function() {
       })
       .then(cleanUp);
   });
+
+  it('should start paused if in a background tab', function() {
+    var target = document.createElement('div');
+    spaniel.__w__.document = { visibilityState: 'hidden' };
+
+    return runTest(
+      {
+        label: 'impression',
+        ratio: 0.5
+      },
+      {
+        target: target,
+        document: { visibilityState: 'hidden' },
+        BACKGROUND_TAB_FIX: true
+      }
+    )
+      .then(function(result) {
+        // Observer should be paused right out of the gate
+        expect(result.observer.paused).to.equal(true);
+
+        // Closing the tab when it never entered the foreground should not trigger any impression events
+        result.observer.onWindowClosed();
+        expect(result.observer.paused).to.equal(true);
+        expect(result.entries.length).to.equal(0);
```

Comment:
do entries get synchronously queued? after onWindowClosed? In other words, can we guarantee that any events that would be fired would cause `entries.length > 0` at this point?

Reply:
Also, does this test fail with the `BACKGROUND_TAB_FIX` flag off?

Reply:
Ok I *think* it's all fired synchronously in the `onWindowClosed()` case, but can you confirm by running the test with the flag off and confirming it fails?

Reply:
Talked to @seanjohnson08 , he already confirmed that the tests fail if the flag isn't on